ossfuzz/decompress_fuzzer.cc

```diff
@@ -0,0 +1,28 @@
+#include <stddef.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include "lz4.h"
+
+#define CHECK(COND)   if (!(COND)) { abort(); }
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+  size_t const buffer_size = 10 * 1024 * 1024;
```

Comment:
Can you add a comment `// TODO: Size input buffer pseudo-randomly based on seed extracted from input`

In the future, I will want to randomly size the buffer based on a seed we extract from the input. Some bugs are only exposed when the output buffer is too small. When the output buffer is too large, we hide potential buffer-overruns beyond the end of the output buffer.

Reply:
Seems like a good plan :smile: